core/crypto/CMakeLists.txt

```diff
@@ -3,6 +3,19 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
+add_library(crypto
```

Comment:
better name is "hasher", since this lib provides only hashing (no enryption/signatures)

Reply:
I was thinking about it, making choice between crypto and hasher. Ok, I can rename.

Reply:
renamed